tests: TestIsCgroupMounted get format error

```
 go test -test.run TestIsCgroupMounted
INFO: switching to fake virtcontainers implementation for testing
INFO: creating test directory
INFO: test directory is /tmp/cc-runtime-809091235
INFO: ensuring required docker image (busybox) is available
INFO: docker image busybox already exists locally
INFO: creating OCI bundle in /tmp/cc-runtime-809091235/bundle for tests to use
--- FAIL: TestIsCgroupMounted (0.00s)
        Error Trace:    oci_test.go:535
	Error:      	Received unexpected error:
	            	stat : no such file or directory
	oci_test.go:539: memory cgroup does not exist: %s memory
FAIL
exit status 1
FAIL	github.com/clearcontainers/runtime	0.550s
```

`oci_test.go:539: memory cgroup does not exist: %s memory` get error format

--- a/oci_test.go
+++ b/oci_test.go
@@ -532,7 +532,7 @@ func TestIsCgroupMounted(t *testing.T) {
 
 	memoryCgroupPath := "/sys/fs/cgroup/memory"
 	if _, err := os.Stat(memoryCgroupPath); os.IsNotExist(err) {
-		t.Skip("memory cgroup does not exist: %s", memoryCgroupPath)
+		t.Skipf("memory cgroup does not exist: %s", memoryCgroupPath)
 	}
 
 	assert.True(isCgroupMounted(memoryCgroupPath), "%s is a cgroup", memoryCgroupPath)

